airflow/providers/amazon/aws/hooks/neptune.py

```diff
@@ -83,3 +89,32 @@ def get_cluster_status(self, cluster_id: str) -> str:
         :return: The status of the cluster.
         """
         return self.get_conn().describe_db_clusters(DBClusterIdentifier=cluster_id)["DBClusters"][0]["Status"]
+
+    def get_db_instance_status(self, instance_id: str) -> str:
+        """
+        Get the status of a Neptune instance.
+
+        :param instance_id: The ID of the instance to get the status of.
+        :return: The status of the instance.
+        """
+        return self.get_conn().describe_db_instances(DBInstanceIdentifier=instance_id)["DBInstances"][0][
+            "DBInstanceStatus"
+        ]
+
+    def wait_for_cluster_instance_availability(
+        self, cluster_id: str, delay: int = 30, max_attempts: int = 60
```

Comment:
Just a small nit here: We've been _trying_ to get standardized names for these types of parameters, and have mostly settled on `waiter_delay` and `waiter_max_attempts`.

Reply:
I'll update it.